### DISCUSSION TITLE
Top bar buttons don't have tooltips

### DISCUSSION THREAD

**Discussion by agentydragon** [athensresearch/athens#948] [Archive]
Hovering mouse on buttons doesn't have a tooltip. Clicking buttons without knowing what they do is a bit "scary" to me as a user.

Also, the "merge roam db" button might be better put somewhere else, it's not a function people will probably use frequently enough to warrant it being always visible.

(I think the same could be argued about the "open database" button)

**Comment by jefftangx** [comment]
@shanberg started work on tooltips, and also thought about designs for moving 1. db picker and 2. merge roam from app toolbar to settings page.

**Comment by jsmorabito** [comment]
resolved in #994 thanks for the post @agentydragon ☺
